sql/testdata/select_non_covering_index

```diff
@@ -63,3 +63,25 @@ EXPLAIN SELECT * FROM t WHERE c > 0 ORDER BY c
 0 index-join
 1 scan       t@c /1-
 1 scan       t@primary
+
+statement ok
+INSERT INTO t VALUES (7, 8, NULL)
+
+query I
+SELECT c FROM t WHERE a=7
+----
+NULL
```

Comment:
Ah, you're right. It's not a non-covering index (although it is a column that is stored in the body rather than making up part of the key). So the test is miscategorized. The panic happens whether the secondary indexes are there or not (as demonstrated by the `boolean_table` test below, which I meant to remove before submitting this PR).